edgexfoundry/device-mqtt-go
# feat!: updates for common config

BREAKING CHANGE: configuration file changed to remove common config settings







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-mqtt-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions

1. run edgex non-secure stack
3. build device mqtt - `make build`
4. run device mqtt using config provider
5. verify that private configuration is loaded in consul
6. ensure that writable in the private config is modifyable - change log level and see change in the terminal

## New Dependency Instructions (If applicable)
